src/extension/ynab-toolkit.css

```diff
@@ -23,6 +23,7 @@ body.theme-dark {
   --tk-default-font-color: #000000;
   --tk-positive-currency-color: #6d9f38;
   --tk-zero-currency-color: #8e8e93;
+  --table_header_background_default: #343437;
```

Comment:
this overrides the value of `table_header_background_default` just for installing the toolkit which isn't something we should do. we only want to set this value if a specific feature is enabled

Reply:
I set it up this way so you could specifically pick which way you wanted it to work.  I reverted the second commit and it now only impacts the way the dark mode looks when highlight rows is enabled.